tests/integration/customerc20.test.ts

```diff
@@ -53,6 +54,7 @@ const withdrawalAmount = BigNumber.from(10)
 describe('Custom ERC20', () => {
   beforeEach('skipIfMainnet', async function () {
     await skipIfMainnet(this)
+    await skipIfNon18Decimals(this)
```

Comment:
Not sure about this one. `registerCustomToken` doesn't seem to support non 18 decimals but it's got nothing to do with the native currency.

Should we make `registerCustomToken` work with non-18 decimals with this PR, or leave as is?

Reply:
Also it's skipped because we're passing tokens based on native currency in tests so it's failing decimals.

Reply:
**TBC: I need to go through it again to confirm the above**

Reply:
Registering custom token is sending retryable ticket to child chain. Tickets needs to be paid for in fee token. Amount of fee token needs to be calculated (and potentially scaled) and provided to the smart contract call. Here's one example: https://github.com/OffchainLabs/token-bridge-contracts/blob/657972fca20cea08b8ac118e6344af0584a3b5b6/test-e2e/orbitTokenBridge.ts#L350